cypress/integration/zzz_setup_portal_remove_user_accounts.spec.js

```diff
@@ -0,0 +1,50 @@
+import * as c from '../support/constants.js'
+import adminPageElements from '../support/elements/admin_page_elements.js'
+import adminSettingsUsersPageElements from '../support/elements/admin_settings_users_page_elements.js'
+import flashNoticePageElements from '../support/elements/flash_notice_page_elements.js'
+import userHomePageElements from '../support/elements/user_home_page_elements.js'
+
+const STUDENT_FULLNAME = c.STUDENT_FIRSTNAME + " " + c.STUDENT_LASTNAME;
+const STUDENT_ONE_FULLNAME = c.STUDENT_ONE_FIRSTNAME + " " + c.STUDENT_ONE_LASTNAME;
+
+// Note for db tracking : This test removes 2 students from the db
+
+context("Teardown : Remove student accounts added at the start of tests", () => {
```

Comment:
I agree that this isn't an elegant solution. The ideal solution to this is to add the setup and teardown of the db and import/publish of the materials into support/index.js which would then run before and after all the tests. The issue with this approach is that because lara is a different super domain in dev and we would actually have to 'visit' lara during this setup and then 'visit' the portal for the actual tests, this isn't going to work out because we need the CSRF token for which we need the UI, for which we need to actually visit lara. That is why I had to do this setup as a separate test and name it this way so that it runs before everything else. I'm also exploring more to see if we can make it the proper way; for now, this is working and we aren't blocked by it.